various-receipes/nestjs/main.ts

```diff
@@ -0,0 +1,18 @@
+import { INestApplication, ValidationPipe } from "@nestjs/common";
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app/app.module";
+
+let app: INestApplication;
+
+export const initializeWebServer = async () => {
+  app = await NestFactory.create(AppModule);
+
+  // ➿ Port is required in Nest.js, we can send 0 to get a available dynamic port
+  await app.listen(0);
+
+  return app;
```

Comment:
@mikicho Not critical, your call, I would suggest returning the address/port like we do with the main example (zero coupling to the internals)